backend/app/tests/api/routes/scheduled_inquiry/test_get_scheduled_inquiries_api.py

```diff
@@ -0,0 +1,60 @@
+from collections.abc import Generator
+
+import pytest
+from fastapi.testclient import TestClient
+from sqlmodel import Session
+
+from app.core.config import settings
+from app.models import Inquiry, InquiryCreate
+from app.services import inquiries as inquiries_service
+from app.services import scheduled_inquiries as scheduled_inquiries_service
+
+route_prefix = "scheduledinquiries"
+
+
+# Populate the db with two test inquiries
+@pytest.fixture
+def db_inquiries(db: Session) -> Generator[tuple[Inquiry, Inquiry]]:
+    # Setup
+    inquiry_in_1 = InquiryCreate(text="Test inquiry 1.")
+    inquiry_in_2 = InquiryCreate(text="Test inquiry 2.")
+    db_inquiry_1 = inquiries_service.create_inquiry(session=db, inquiry_in=inquiry_in_1)
+    db_inquiry_2 = inquiries_service.create_inquiry(session=db, inquiry_in=inquiry_in_2)
+
+    yield db_inquiry_1, db_inquiry_2
+
+    # Teardown
+    db.delete(db_inquiry_1)
+    db.delete(db_inquiry_2)
+
+
+def test_get_scheduled_inquiries_when_scheduled_inquiries_exist_should_return_all(
```

Comment:
just wondering, should there be a case with an empty scheduled inquiries list?

Reply:
Caught me being lazy